djangoevents/tests/test_unifiedtranscoder.py

```diff
@@ -14,6 +14,12 @@ class Updated(EventSourcedEntity.AttributeChanged):
         event_type = 'overridden_event_type'
 
 
+def adds_schema_version_to_event_data(adds):
```

Comment:
Please rename to "apply_adding_schema_version" or similar that will look like a "command"